### PR TITLE
Templates: Add new templates from batch 3

### LINKED ISSUE #7236: Templates Project: Batch 3 (21-30)
## Task Description

[**Figma link**](https://www.figma.com/file/OQIT5ISzdQVvRD7Aor8ajA/Web-Stories-50-Templates?node-id=1682%3A0)

Screenshot 2021-04-16 at 17 48 14

This batch includes 10 (9?) new templates:

1. Entertainment
2. Entertainment
3. Travel
4. Travel
5. Technology
6. Technology
7. Fashion & Beauty
8. Arts & Craft
9. Home & Garden

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Summary

Adds 9 new story templates.

- Rock Music Festival
- Celebrity Q & A
- Los Angeles City Guide
- Hawaii Travel Packing List
- Google Music Studio Tour
- How Contact Tracing Works
- Summer Fashion Collection
- Buying Art on the Internet
- House Hunting

Meta data for the templates were provided along with the designs in the Figma file itself, this includes sticker names, title, tags, description and colour palette.

## Relevant Technical Choices

- Adds stickers and story template JSON
- Uses title/alt texts for videos and images from the original source i.e. getty / unsplash / pexels.

## Testing Instructions

New templates should be available under 'Explore Templates' page.
`/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery`

Screenshot 2021-08-02 at 1 48 55 PM

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7236
Companion PR #8526

### CODE REVIEW

File: packages/stickers/src/how-contact-tracing-works/phone-in-hand.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
```
Comment (swissspidy) [inline packages/stickers/src/how-contact-tracing-works/phone-in-hand.js]: Perhaps rename file to `phoneInHand` for consistency in file naming?

File: packages/stickers/src/summer-fashion-collection/yellowHeart.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { _x } from '@web-stories-wp/i18n';
+import PropTypes from 'prop-types';
+
+const title = _x('YellowHeart', 'sticker name', 'web-stories');
```
Comment (swissspidy) [inline packages/stickers/src/summer-fashion-collection/yellowHeart.js]:
```suggestion
const title = _x('Yellow Heart', 'sticker name', 'web-stories');
```

File: packages/templates/src/raw/buying-art-on-the-internet/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  tags: [
+    _x('Arts & Craft', 'template keyword', 'web-stories'),
+    _x('Paint', 'template keyword', 'web-stories'),
+    _x('Tips', 'template keyword', 'web-stories'),
+    _x('Clean', 'template keyword', 'web-stories'),
+    _x('White', 'template keyword', 'web-stories'),
+  ],
+  colors: [
+    {
+      label: _x('International Klein Blue', 'color', 'web-stories'),
```
Comment (swissspidy) [inline packages/templates/src/raw/buying-art-on-the-internet/index.js:38]: TIL about this color 😄

File: packages/templates/src/raw/celebrity-q-and-a/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ * Internal dependencies
+ */
+import { default as template } from './template';
+
+export default {
+  title: _x('Celebrity Q & A', 'template name', 'web-stories'),
+  tags: [
+    _x('Entertainment', 'template keyword', 'web-stories'),
+    _x('Story', 'template keyword', 'web-stories'),
```
Comment (swissspidy) [inline packages/templates/src/raw/celebrity-q-and-a/index.js]: This keyword seems redundant... Obviously it's a (web) story.

Comment (divyarajmasani) [reply packages/templates/src/raw/celebrity-q-and-a/index.js]: Could it be that the word `Story` here may not be indicative of a Web Story but actual story, like story, tale telling. Not sure though 😅 . Can be removed as seems ambiguous.

Comment (swissspidy) [reply packages/templates/src/raw/celebrity-q-and-a/index.js]: Hmm yeah problably. Perhaps replace with `Interview`?

**Review by swissspidy — APPROVED** [review]

**Pull request merged**